NGitLab.Tests/Docker/GitLabDockerContainer.cs

```diff
@@ -320,10 +320,11 @@ async Task GenerateAdminToken(GitLabCredential credentials)
 
                 await page.GotoAsync(GitLabUrl + "/-/profile/personal_access_tokens");
 
-                var formLocator = page.Locator("main#content-body form");
+                await page.Locator("main[id='content-body'] button[data-testid='add-new-token-button']").ClickAsync();
+                var formLocator = page.Locator("main[id='content-body'] form[id='js-new-access-token-form']");
 
                 var tokenName = "GitLabClientTest-" + DateTime.UtcNow.ToString("yyyyMMdd-HHmmss", CultureInfo.InvariantCulture);
-                await formLocator.GetByLabel("Token name").FillAsync(tokenName);
+                await formLocator.Locator("input[data-testid='access-token-name-field']").FillAsync(tokenName);
```

Comment:
To initialize the GitLab server for versions 15 and 16, the following code seems to work _(though probably not optimal)_:

```suggestion
                var tokenName = "GitLabClientTest-" + DateTime.UtcNow.ToString("yyyyMMdd-HHmmss", CultureInfo.InvariantCulture);

                await page.GotoAsync(GitLabUrl + "/-/profile/personal_access_tokens");

                ILocator formLocator;
                try
                {
                    await page.Locator("main[id='content-body'] button[data-testid='add-new-token-button']").ClickAsync(new LocatorClickOptions { Timeout = 5_000 });
                    formLocator = page.Locator("main[id='content-body'] form[id='js-new-access-token-form']");
                    await formLocator.Locator("input[data-testid='access-token-name-field']").FillAsync(tokenName);
                }
                catch
                {
                    // Try the "old" 15.x.y way
                    await page.GotoAsync(GitLabUrl + "/-/profile/personal_access_tokens");
                    formLocator = page.Locator("main#content-body form");
                    await formLocator.GetByLabel("Token name").FillAsync(tokenName);
                }
```

Reply:
Proposal committed
[fcdaa4f](https://github.com/ubisoft/NGitLab/pull/682/commits/fcdaa4fbb86fd5be718c7f9121d37a2963f2c5b9)